en/app.ftl

```diff
@@ -278,6 +278,25 @@ landing-use-cases-signups-hero-content2 = Before you complete that next signup,
 landing-faq-headline = Top questions about { -brand-name-firefox-relay }
 landing-faq-cta = See more FAQs about { -brand-name-firefox-relay }
 
+## Landing Reviews Section
+
+landing-reviews-add-ons = Add-ons
+landing-reviews-logo-title = { -brand-name-firefox } Browser
+# Do not localize addons.mozilla.org
+landing-reviews-details-source = Source: addons.mozilla.org
+landing-reviews-rating = out of 5 ({ $review_count } reviews)
```

Comment:
The string seems to be missing a variable before `out of 5` which refers to the rating. `{ $review_count } reviews` refers to the number of reviews.

Reply:
The rating here is a separate element, can we proceed without it?

Reply:
To clarify, this is what it looks like:

![image](https://user-images.githubusercontent.com/4251/172846060-8a231fff-dd33-433a-b70a-262e1ebd9fa0.png)

(can be seen [here](https://deploy-preview-2030--fx-relay-demo.netlify.app/))

I'm not sure if we'd be able to deal with it if a locale would e.g. put text in front of the number, technically. But I suppose we should at least add a comment to help localisers? Something like:

```suggestion
# This string is displayed in a smaller font under a big number representing the average review score (e.g. "4.2").
landing-reviews-rating = out of 5 ({ $review_count } reviews)
```

Also, @lloan, this does make me realise that the way fractional numbers are formatted also differs across languages (i.e. it'd be `4,2` in Dutch). You might want to wrap that in [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) i.c.w. [`getLocale`](https://github.com/mozilla/fx-private-relay/blob/main/frontend/src/functions/getLocale.ts).

Reply:
Thank you for adding some context here. We will go with the current solution and see if have any feedback on the approach.